src/Symfony/Component/AssetMapper/ImportMap/ImportMapRenderer.php

```diff
@@ -70,7 +73,9 @@ public function render(string|array $entryPoint, array $attributes = []): string
             }
 
             $preload = $data['preload'] ?? false;
-            if ('css' !== $data['type']) {
+            if ('json' === $data['type']) {
+                $importMap[$importName] = 'data:application/javascript,'.str_replace('%', '%25', \sprintf(self::LOADER_JSON, addslashes($path)));
```

Comment:
```suggestion
                $importMap[$importName] = $path;
```

I think it should be sufficient (I still need to test it) given my last comment.

Reply:
Yeah it should be `$path`, see https://github.com/symfony/symfony/pull/61133#issuecomment-3077593477